recipes/git-cliff/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.0.2" %}
+
+package:
+  name: git-cliff
+  version: {{ version }}
+
+source:
+  url: https://github.com/orhun/git-cliff/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: ab29fbda532bec2a344d99f39b7998063c576efee92a1bee083c6fba5f52e4ef
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - pkg-config
+    - cargo-bundle-licenses
+
+  host:
+    - libgit2
+
+test:
+  commands:
+    - git-cliff --help
+
+about:
+  home: https://git-cliff.org
+  summary: "A highly customizable changelog generator"
+  description: "A highly customizable Changelog Generator that follows Conventional Commit specifications ⛰️"
+  license: MIT OR Apache-2.0
+  license_file: 
+    - LICENSE-MIT
+    - LICENSE-APACHE
+    - THIRDPARTY.yml
+  doc_url: https://git-cliff.org/docs/
+  dev_url: https://github.com/orhun/git-cliff
+
+extra:
+  recipe-maintainers:
+    - orhun
```

Comment:
@wolfv Do you also want to add yourself here?

Reply:
I think it's fine for the moment :)